### PR TITLE
UI for the Sync auto-config feature

### PR DESCRIPTION
**Pull request by LunarX**
The bottom sheet for discovering the feature is missing and will be added in a second PR

### CODE REVIEW

**Review by FabianDevel — CHANGES_REQUESTED** [review]
This will go to playstore even with fDroid, we surely don't want that

File: app/src/main/java/com/infomaniak/mail/ui/sync/SyncOnboardingFragment.kt [hunk 1]
```diff
@@ -39,18 +40,20 @@ class SyncOnboardingFragment : Fragment() {
...
         setupClickListener()
     }
 
     private fun setupClickListener() {
-        val (direction, trackerName) = if (syncAutoConfigViewModel.isSyncAppUpToDate()) {
-            SyncOnboardingFragmentDirections.actionSyncOnboardingFragmentToSyncStartFragment() to "openStart"
-        } else {
-            SyncOnboardingFragmentDirections.actionSyncOnboardingFragmentToSyncInstallFragment() to "openInstall"
-        }
+        val trackerName = if (syncAutoConfigViewModel.isSyncAppUpToDate()) "openStart" else "openInstall"
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/ui/sync/SyncOnboardingFragment.kt:53]: Check all Matomo for this feature, the names changed.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/res/navigation/sync_auto_config_navigation.xml [hunk 2]
```diff
@@ -29,25 +29,12 @@
...
-        <action
-            android:id="@+id/action_syncInstallFragment_to_syncStartFragment"
-            app:destination="@id/syncStartFragment" />
-    </fragment>
-
-    <fragment
-        android:id="@+id/syncStartFragment"
         android:name="com.infomaniak.mail.ui.sync.SyncStartFragment"
-        android:label="SyncStartFragment"
+        android:label="SyncInstallFragment"
```
Comment (KevinBoulongne) [inline app/src/main/res/navigation/sync_auto_config_navigation.xml:37]: The names are not consistent.

Rename this fragment as `SyncConfigureFragment`.

**Review by FabianDevel — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_sync_onboarding.xml [hunk 3]
```diff
@@ -15,27 +15,113 @@
...
+        android:id="@+id/wave"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="0dp"
+        android:importantForAccessibility="no"
+        android:scaleType="fitXY"
+        android:src="@drawable/ic_back_wave_sync"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintHorizontal_bias="0.0"
```
[the remark this answers is not in the document]
Comment (LunarX) [reply app/src/main/res/layout/fragment_sync_onboarding.xml:34]: Definitely not

File: app/src/main/res/drawable/illustration_discover_sync.xml [hunk 4]
```diff
@@ -0,0 +1,230 @@
+<!--
```
Comment (KevinBoulongne) [inline app/src/main/res/drawable/illustration_discover_sync.xml:1]:
```suggestion
<?xml version="1.0" encoding="utf-8"?><!--
```

File: app/src/main/res/layout/activity_sync_auto_config.xml [hunk 5]
```diff
@@ -22,20 +22,13 @@
     android:layout_height="match_parent"
     tools:context=".ui.sync.SyncAutoConfigActivity">
 
-    <LinearLayout
+
```
Comment (KevinBoulongne) [inline app/src/main/res/layout/activity_sync_auto_config.xml]:
```suggestion
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Review by FabianDevel — APPROVED** [review]

**Pull request merged**